.claude/commands/propose-risk.md

```diff
@@ -0,0 +1,427 @@
+---
+description: Propose an upgrade risk based on a Jira issue and existing risks
+---
+
+You are helping create a new upgrade risk declaration based on a Jira issue.
+
+## Arguments
+
+Usage: `/propose-risk <jira-issue>`
+
+- `$1` - Jira issue ID (e.g., `MCO-1834`, `CORENET-6196`, `OCPBUGS-12345`)
+
+Example: `/propose-risk MCO-1834`
+
+## Task
+
+Your goal is to propose a complete, well-formed upgrade risk YAML file for the blocked-edges directory based on the Jira issue and existing patterns in the codebase.
+
+### Step 1: Research the Jira Issue
+
+1. Extract the Jira issue URL:
+   - For RedHat Jira: `https://issues.redhat.com/browse/$1`
+   - For OCPBUGS: `https://issues.redhat.com/browse/$1`
```

Comment:
Same advice for each option?  Maybe consolidate to not have to bother with the Jira-vs.-OCPBUGS distinction?

Reply:
Thanks, done.